bundles/org.openhab.binding.renault/README.md

```diff
@@ -0,0 +1,45 @@
+# Renault Binding
+
+This binding allow MyRenault App. users to get battery status and other data from their cars.
+
+A binding that translates the [python based renault-api](https://renault-api.readthedocs.io/en/latest/) in an easy to use binding. 
+
+
+## Supported Things
+
+Works on my car (Renault Zoe 50) but I only have one car to test.
+
+
+## Discovery
+
+No discovery
+
+## Binding Configuration
+
+You require your MyRenault credential, locale and VIN for your MyRenault registered car.
```

Comment:
There is no binding configuration. This sentence has to be moved to the next section (thing configuration).

Reply:
Done.